src/features/recruitments/components/filter-panel.tsx

```
@@ -0,0 +1,124 @@
+  trigger: React.ReactNode;
+  footer: React.ReactNode;
+  children: React.ReactNode;
+  contentClassName?: string;
+}
+
+function CloseButton({ onClick }: { onClick: () => void }) {
+  return (
+    <button
+      onClick={onClick}
+      className="text-muted-foreground hover:text-foreground cursor-pointer rounded-md p-1 transition-colors"
+    >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)